primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -29,116 +28,115 @@ Style: Signs,Arial,20,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
+Dialogue: 0,0:08:16.90,0:08:19.21,Default,,0,0,0,,Are you fine sending her out there?
 Dialogue: 0,0:08:19.21,0:08:20.43,Default,,0,0,0,,What's that supposed to mean?
-Dialogue: 0,0:08:20.43,0:08:23.18,Default,,0,0,0,,I don't let personal feelings interfere with work!
-Dialogue: 0,0:08:23.84,0:08:25.85,Default,,0,0,0,,Then make me take point!
+Dialogue: 0,0:08:20.43,0:08:23.18,Default,,0,0,0,,I don't let my personal feelings \Nget in the way of work!
+Dialogue: 0,0:08:23.84,0:08:25.85,Default,,0,0,0,,Then put me on point!
 Dialogue: 0,0:08:25.85,0:08:28.28,Default,,0,0,0,,I'm superior to Rin in all areas.
-Dialogue: 0,0:08:28.28,0:08:29.35,Default,,0,0,0,,I can accomplish this task.
+Dialogue: 0,0:08:28.28,0:08:29.35,Default,,0,0,0,,I can easily do the job.
 Dialogue: 0,0:08:29.35,0:08:31.47,Default,,0,0,0,,You act on your own authority.
 Dialogue: 0,0:08:31.47,0:08:33.53,Default,,0,0,0,,You're a loose cannon.
 Dialogue: 0,0:08:37.07,0:08:38.86,Default,,0,0,0,,So that's it.
 Dialogue: 0,0:08:38.86,0:08:41.77,Default,,0,0,0,,That's why I'm inferior to Rin, is it?
 Dialogue: 0,0:08:42.57,0:08:43.57,Default,,0,0,0,,Misaki...
-Dialogue: 0,0:08:44.77,0:08:46.24,Default,,0,0,0,,I'll show him.
-Dialogue: 0,0:08:47.07,0:08:49.22,Default,,0,0,0,,I'll {\i1}prove{\i0} I'm superior to Rin.
+Dialogue: 0,0:08:44.77,0:08:46.24,Default,,0,0,0,,{\i1}I'll show him.
+Dialogue: 0,0:08:47.07,0:08:49.22,Default,,0,0,0,,{\i1}I'll {\i0}prove{\i1} I'm superior to Rin.
 Dialogue: 0,0:08:53.61,0:08:58.34,Default,,0,0,0,,You're on point. Ensure you penetrate the target directly with the probe.{？？に装填された探査弾を？？確実に打ち込んで下さい - you can see the scene from the vn here: https://youtu.be/dkFsJWiJVQ8?t=1429  but the audio is almost as inaudible there as here}{j - I can definitely hear ファースト after a relisten}
 Dialogue: 0,0:08:58.34,0:09:00.72,Default,,0,0,0,,With this improvised weapon?
 Dialogue: 0,0:09:00.72,0:09:02.99,Default,,0,0,0,,01 Skelter Wave, copy that.
 Dialogue: 0,0:09:21.37,0:09:23.85,Default,,0,0,0,,Skelter Wave, preparing for launch.
 Dialogue: 0,0:09:23.85,0:09:24.78,Default,,0,0,0,,Roger!
 Dialogue: 0,0:09:25.05,0:09:27.58,Default,,0,0,0,,{\i1}Misaki's completely right.
 Dialogue: 0,0:09:27.58,0:09:30.31,Default,,0,0,0,,{\i1}She'd do a better job for sure.
-Dialogue: 0,0:09:30.31,0:09:33.72,Default,,0,0,0,,{\i1}But I want to live up to Captain Funagai's expectations.
+Dialogue: 0,0:09:30.31,0:09:33.72,Default,,0,0,0,,{\i1}But I want to live up \Nto Captain Funagai's expectations.
 Dialogue: 0,0:09:33.72,0:09:35.99,Default,,0,0,0,,{\i1}No one's going to listen to excuses.
-Dialogue: 0,0:09:35.99,0:09:38.63,Default,,0,0,0,,{\i1}I won't give in to the enemy or my own weakness.
-Dialogue: 0,0:09:38.63,0:09:41.16,Default,,0,0,0,,{\i1}Then I'll tell him how I feel!{literally just "convey to him", but this seems an obvious enough inference}
+Dialogue: 0,0:09:35.99,0:09:38.63,Default,,0,0,0,,{\i1}I won't give in to the enemy \Nor my own weakness.
```

Comment:
There's a pause in the dialogue that lends itself nicely to an ellipsis (or a comma, but I think an ellipsis works better).
```suggestion
Dialogue: 0,0:09:35.99,0:09:38.63,Default,,0,0,0,,{\i1}I won't give in... not to the enemy, \Nnor my own weakness.
```